Sming/Makefile

```diff
@@ -140,6 +140,14 @@ cs: ##Apply coding style to all core files
 		clang-format -i -style=file $$FILE; \
 	done
 
+ALL_FILES = $(filter %.cpp %.h %.c %.mk %.sh %.cmd,$(shell $(GIT) ls-files))
+
+.PHONY: dos2unix
+dos2unix: ##Make sure all our files use Unix line endings
+	@dos2unix $(ALL_FILES)
```

Comment:
Please, use `$(Q) dos2unix ...` instead of `@dos2unix`. Same for the line below.

Reply:
OK, have done that.